OmgvaPOS/src/OrderItemManagement/Service/OrderItemDeletionService.cs

```diff
@@ -0,0 +1,97 @@
+﻿using OmgvaPOS.Database.Context;
+using OmgvaPOS.ItemManagement.Repositories;
+using OmgvaPOS.ItemManagement.Validators;
+using OmgvaPOS.ItemVariationManagement.Models;
+using OmgvaPOS.ItemVariationManagement.Repositories;
+using OmgvaPOS.ItemVariationManagement.Validators;
+using OmgvaPOS.OrderItemManagement.Repository;
+using OmgvaPOS.OrderItemManagement.Validators;
+using OmgvaPOS.OrderManagement.Repository;
+using OmgvaPOS.OrderManagement.Validators;
+
+namespace OmgvaPOS.OrderItemManagement.Service
+{
+    public class OrderItemDeletionService: IOrderItemDeletionService
+    {
+        private readonly OmgvaDbContext _context;
+        private readonly IOrderRepository _orderRepository;
+        private readonly IOrderItemRepository _orderItemRepository;
+        private readonly IItemRepository _itemRepository;
+        private readonly IItemVariationRepository _itemVariationRepository;
+        private readonly ILogger<OrderItemDeletionService> _logger;
+        public OrderItemDeletionService(
```

Comment:
This thing should be put in a constructor in the class. Reason - it's huge and not really readable.